### DISCUSSION TITLE
Possible to build FullpageOS as arm64 i.e. 64-bit?

### DISCUSSION THREAD

**Discussion by shaunhurley** [guysoft/FullPageOS#557] [Q&A] [answered]
Hi Folks,

Lots of backstory that I doubt anyone would care about, but the tl;dr; question is:  can FullpageOS be built against the arm64 / Debian 12.5 Raspian image, versus the armhf / Debian 12.1 OS image?

I'm trying to do some other (full disclosure - not FullpageOS related :S) stuff with puppeteer and chromium and am having some incompatibility issues on the armhf build :(

Thanks!

Shaun

**Comment by guysoft** [comment, marked as answer]
Yes sure. But ATM I dont have time to maintain it.
It should work without major changes, there are env settings to usr arm64 on CustomPiOS. I am working on CustomPiOS v2 that should suport a bunch of boards easily. But its still in alpha.

**Comment by shaunhurley** [comment]
Thanks for that, I managed to work around the need and get things working on the 32-Bit daily bookworm build.  I am now curious though, so when I get a moment I might have a play :)

**Reply by chriskewl** [reply]
Were you able to make this work? I am looking to use arm64 for the 10% speed boost. Thanks!
